types/ember__component/index.d.ts

```diff
@@ -8,10 +8,11 @@
 
 /// <reference types="jquery" />
```

Comment:
Huh, looks like I missed this doing the v4 cleanup. I'll do a follow-on PR for it.